src/Component/Navbar/Navbar.css

```diff
@@ -38,20 +39,20 @@
   color: var(--dark-cyan);
   cursor: pointer;
 }
-@media only screen and (min-device-width: 320px) and (max-device-width: 880px) {
+@media only screen and (min-device-width: 320px) and (max-device-width: 880px) and (-webkit-min-device-pixel-ratio: 2) {
   .Header-container {
```

Comment:
for classNames use camelCase, PascalCase or lower-case. For example, you can use here : headerContainer or HeaderContainer or Header_Container or header-container.

Reply:
will keep in mind